benches/benchmark.rs

```diff
@@ -83,14 +86,15 @@ fn generate_crl(revoked_count: usize) -> Vec<u8> {
         this_update: date_time_ymd(2023, 6, 17),
         next_update: date_time_ymd(2024, 6, 17),
         crl_number: SerialNumber::from(1234),
-        alg: &PKCS_ECDSA_P256_SHA256,
         key_identifier_method: KeyIdMethod::Sha256,
         issuing_distribution_point: None,
         revoked_certs,
     };
-    let crl = CertificateRevocationList::from_params(crl).unwrap();
     let issuer = CRL_ISSUER.lock().unwrap();
-    crl.serialize_der_with_signer(&issuer).unwrap()
+    crl.signed_by(&issuer.cert, &issuer.key_pair)
```

Comment:
Should `generate_crl()` actually yield a `CertificateRevocationListDer<'static>` now (`crl.signed_by(..).unwrap().into()`)?

Reply:
Good idea. Done.